tox.ini

```diff
@@ -16,7 +15,8 @@ env =
 
 [testenv:lint]
 deps =
-    -e .
+    -e .[proxy]
+    dnspython
```

Comment:
Why is this required? 🤔

Reply:
I've added them so mypy could check types. This isn't useful (currently) for proxy since it's done have typing.
Bit It help with dnspython for checking type with call to `dns.resolver`